### PR TITLE
Added lua style shortcut example for reach

### PR DESCRIPTION
**Pull request by Nacho114**
- Gave example of lua style shortcut for reach, based on this [discussion](https://github.com/toppair/reach.nvim/issues/14#issuecomment-1320924962)
- Added vim-plug based usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,16 @@ buffer / mark / tabpage / colorscheme switcher for [Neovim](https://github.com/n
 
 ### Installation
 
+Using lua
```
Comment (toppair) [inline README.md:13]:
```suggestion
Using [packer.nvim](https://github.com/wbthomason/packer.nvim)

```

File: README.md [hunk 2]
```diff
@@ -10,10 +10,16 @@ buffer / mark / tabpage / colorscheme switcher for [Neovim](https://github.com/n
...
 ```lua
 use 'toppair/reach.nvim'
 ```
 
+Using [vim-plug](https://github.com/junegunn/vim-plug)
```
Comment (toppair) [inline README.md:18]:
```suggestion
Using [vim-plug](https://github.com/junegunn/vim-plug)

```

File: README.md [hunk 3]
```diff
@@ -70,6 +76,9 @@ local options = {
 }
 
 require('reach').buffers(options)
+
+-- Enable reach buffers using leader keymap
```
Comment (toppair) [inline README.md:80]:
```suggestion
-- Example keymapping
```

**Review by toppair — CHANGES_REQUESTED** [review]
Thanks. Left some suggestions., apply them and it'll be good to merge.

**Comment by Nacho114** [comment]
> Thanks. Left some suggestions., apply them and it'll be good to merge.

Done 👍🏼

**Review by toppair — APPROVED** [review]

**Pull request merged**